src/cycleclock.h

```diff
@@ -164,6 +164,21 @@ inline BENCHMARK_ALWAYS_INLINE int64_t Now() {
   uint64_t tsc;
```

Comment:
Not directly related to this patch, but this preprocessor hell is such a maze :/
This needs refactoring into separate functions.